index.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * Plugin Name:       The Events Calendar Extension: Divi Theme Compatibility
+ * Plugin URI:        https://theeventscalendar.com/extensions/elegant-themes-divi-theme-compatibility/
+ * GitHub Plugin URI: https://github.com/mt-support/tribe-ext-divi-theme-compatibility
+ * Description:       Makes The Events Calendar compatible with Elegant Themes' Divi theme and Divi-based themes (e.g. Extra theme). The posts_per_page / pagination fix should also work for all their themes, even if not Divi-based.
+ * Version:           1.1.0
+ * Extension Class:   Tribe__Extension__Divi_Theme_Compatibility
+ * Author:            Modern Tribe, Inc.
+ * Author URI:        http://m.tri.be/1971
+ * License:           GPL version 3 or any later version
+ * License URI:       https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ *     This plugin is free software: you can redistribute it and/or modify
+ *     it under the terms of the GNU General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     any later version.
+ *
+ *     This plugin is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *     GNU General Public License for more details.
+ */
+
+// Do not load unless Tribe Common is fully loaded and our class does not yet exist.
+if (
+    class_exists('Tribe__Extension')
+    && !class_exists('Tribe__Extension__Example')
```

Comment:
Whitespace (2/3): there should be a space after any usage of the bang operator (ie `! class_exists( '' )`)

Reply:
Whitespace (3/3): here and throughout the change, we should have spaces after and before the opening/closing parentheses (ie `do_func( 'foo', $bar )`)